commit 2e14a3ac178ee87aa9154e5a15dcd986af1b6059
Author: Stephan Hartmann <stha09@googlemail.com>
Date:   Tue Mar 28 14:34:55 2023 +0000

    IWYU: add cstring for std::strlen in web_view_impl.cc
    
    Bug: 957519
    Change-Id: I15ad1e905eda3d96bbf164f5df8f3cf8e02bc370
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4376772
    Reviewed-by: Alex Ilin <alexilin@chromium.org>
    Commit-Queue: Alex Ilin <alexilin@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1123011}

diff --git a/chrome/test/chromedriver/chrome/web_view_impl.cc b/chrome/test/chromedriver/chrome/web_view_impl.cc
index f726627e88def..d96d481d0bc40 100644
--- a/chrome/test/chromedriver/chrome/web_view_impl.cc
+++ b/chrome/test/chromedriver/chrome/web_view_impl.cc
@@ -6,6 +6,7 @@
 
 #include <stddef.h>
 #include <algorithm>
+#include <cstring>
 #include <memory>
 #include <queue>
 #include <utility>
